### PR TITLE
Adding generic login page design

### PR DESCRIPTION
**Pull request by MartoPavlov**

### CODE REVIEW

**Comment by kvenelinov** [comment]
Could you attach a snip?

File: client/src/componets/Login.tsx [hunk 1]
```diff
@@ -1,11 +1,110 @@
...
+            alert(JSON.stringify(values, null, 2));
+        }
+    });
+
+    const styles = useStyles();
+    
+    return (
+        <div className={styles.root}>
+            <form className={styles.formContainer} onSubmit={formik.handleSubmit}>
+                <h1 className={styles.title}>Login {formik.submitCount}</h1>
```
Comment (mihaelagandeva) [inline client/src/componets/Login.tsx]: Този count защо ни е ?

**Comment by MartoPavlov** [comment]
Snip?

File: client/src/componets/Login.tsx (hunk 1 above)
Comment (MartoPavlov) [reply client/src/componets/Login.tsx]: Ops. I must have forgotten to remove it.

**Comment by MartoPavlov** [comment]
12ec6413e82b5f0fa77fbb9bc3211ea8

**Review by kvenelinov — APPROVED** [review]

**Pull request merged**